.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -8,52 +8,7 @@ body:
       value: |
         Thank you for reporting the problem!
         Please make sure what you are reporting is a bug with reproducible steps. To ask questions
-        or share ideas, pleae post on our [Discussion page](https://discuss.ray.io/) instead.
-
-  - type: checkboxes
```

Comment:
Removed this since redundant issues are actually a strong signal to us that an issue is important. "Acks" on existing issues are often lost in the noise.

We also don't have a strong problem of duplicate issues for now.